docs/userguide/pyproject_config.rst

```diff
@@ -211,6 +211,14 @@ however please keep in mind that all non-comment lines must conform with :pep:`5
 
 .. rubric:: Notes
 
+.. [#pep660-status] Editable install without ``setup.py`` will be supported from
+   ``setuptools >= 63.0.0``, but this feature will remain experimental for a while.
+   Check https://github.com/pypa/setuptools/issues/2816 for detail.
```

Comment:
Since setuptools 63 was released without this support, I am a bit confused now.

Reply:
Sorry for the confusion. There was a different major change that got in the pipeline before this one.

PEP 660 will probably get in on v64. It is currently under tests.

Reply:
Ah, thank you for pointing it out. I misunderstood the situation.